libr/include/r_util/r_annotated_code.h

```diff
@@ -106,12 +106,34 @@ R_API void r_annotated_code_free(RAnnotatedCode *code);
  * @e: Pointer to the annotation
  * @user: Always NULL for this function. Present here for this function to be of the type RVectorFree.
  * 
- * This function recongnizes the type of the specified annotation and
+ * This function recognizes the type of the specified annotation and
  * frees memory that is dynamically allocated for it.
  * 
  * Return: Nothing.
  */
 R_API void r_annotation_free(void *e, void *user);
+/**
```

Comment:
Please don't invent your own documentation comment style. If you use doxygen style comments do so consistently. They are not only for human reading but also consumption by tools. See the documentation https://www.doxygen.nl/manual/index.html

Reply:
Radare2 uses this style https://www.kernel.org/doc/html/v4.16/doc-guide/kernel-doc.html#how-to-format-kernel-doc-comments
I have been told to use this and all functions in `r_annotated_code.h` are documented using this style.

Reply:
See here: https://github.com/radareorg/radare2/blob/master/libr/include/r_util/r_annotated_code.h#L61

Reply:
Is it written anywhere r2 documentation that it uses this style? If it I am fine with that, but I haven't seen it used anywhere except r_annotated_code.

Reply:
This https://github.com/radareorg/radare2/blob/master/DEVELOPERS.md claims that r2 uses doxygen C-style comments.

Reply:
@ITAYC0HEN @thestr4ng3r I'm not sure when this was decided. See this comment by @ITAYC0HEN https://github.com/radareorg/radare2/pull/16939#pullrequestreview-417269610 It was discussed when I was working on #16939 .

Reply:
Doxygen is the way to go.